.gitignore

```diff
@@ -0,0 +1,2 @@
+test.node
```

Comment:
- [ ] Nice work making use of `.gitignore` :clap: However the `test.md` file is still in your repo. Kindly remove it from your repo, and make sure it ignores by the git system. 
Please note that the file name is `test.md` not `test.node`